### ISSUE TITLE
ERROR: An error occurred during the fetch of repository 'local_config_git'

### ISSUE THREAD

**Issue by mpmischitelli** [tensorflow/serving#1558]
### System information
- Microsoft Windows 10 Enterprise LTSC 1809
- TensorFlow Serving installed from source
- TensorFlow Serving version: current head (b9602b38)
- Using bazelisk with USE_BAZEL_VERSION="1.2.1". Using bazel 1.2.1 directly doesn't solve the issue.

### Describe the problem
Running the following command (taken from Dockerfile.devel) in either powershell or cmd:
```bazel build --color=yes --curses=yes --verbose_failures --output_filter=DONT_MATCH_ANYTHING --config=nativeopt tensorflow_serving/model_servers:tensorflow_model_server```

Fails with the following:
```
INFO: Call stack for the definition of repository 'local_config_git' which is a git_configure (rule definition at C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl:66:17):
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:74:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:61:5
 - C:/git/serving/tensorflow_serving/workspace.bzl:11:5
 - C:/git/serving/WORKSPACE:62:1
ERROR: An error occurred during the fetch of repository 'local_config_git':
   Traceback (most recent call last):
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 64
                _fail(result.stderr)
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
                fail(<1 more arguments>)
Git Configuration Error: java.io.IOException: ERROR: src/main/native/windows/process.cc(199): CreateProcessW("C:\users\<user>\_bazel_<user>\7cok3vt4\external\local_config_git\usr\bin\python3" C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py --configure C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/ --gen_root_path C:/users/<user>/_bazel_<user>/7cok3vt4/external/local_config_git/gen): The system cannot find the file specified.
 (error: 2)
INFO: Call stack for the definition of repository 'farmhash_archive' which is a tf_http_archive (rule definition at C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/repo.bzl:121:19):
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:252:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:61:5
 - C:/git/serving/tensorflow_serving/workspace.bzl:11:5
 - C:/git/serving/WORKSPACE:62:1
INFO: Call stack for the definition of repository 'boringssl' which is a tf_http_archive (rule definition at C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/repo.bzl:121:19):
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:584:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:61:5
 - C:/git/serving/tensorflow_serving/workspace.bzl:11:5
 - C:/git/serving/WORKSPACE:62:1
INFO: Call stack for the definition of repository 'curl' which is a tf_http_archive (rule definition at C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/repo.bzl:121:19):
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:503:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:61:5
 - C:/git/serving/tensorflow_serving/workspace.bzl:11:5
 - C:/git/serving/WORKSPACE:62:1
INFO: Call stack for the definition of repository 'grpc' which is a tf_http_archive (rule definition at C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/repo.bzl:121:19):
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:516:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:61:5
 - C:/git/serving/tensorflow_serving/workspace.bzl:11:5
 - C:/git/serving/WORKSPACE:62:1
INFO: Call stack for the definition of repository 'eigen_archive' which is a tf_http_archive (rule definition at C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/repo.bzl:121:19):
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:171:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:61:5
 - C:/git/serving/tensorflow_serving/workspace.bzl:11:5
 - C:/git/serving/WORKSPACE:62:1
INFO: Call stack for the definition of repository 'aws' which is a third_party_http_archive (rule definition at C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/repo.bzl:201:28):
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/aws/workspace.bzl:9:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:40:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:82:5
 - C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/workspace.bzl:61:5
 - C:/git/serving/tensorflow_serving/workspace.bzl:11:5
 - C:/git/serving/WORKSPACE:62:1
ERROR: C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/core/BUILD:2494:1: @org_tensorflow//tensorflow/core:version_info_gen depends on @local_config_git//:gen/spec.json in repository @local_config_git which failed to fetch. no such package '@local_config_git//': Traceback (most recent call last):
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 64
                _fail(result.stderr)
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
                fail(<1 more arguments>)
Git Configuration Error: java.io.IOException: ERROR: src/main/native/windows/process.cc(199): CreateProcessW("C:\users\<user>\_bazel_<user>\7cok3vt4\external\local_config_git\usr\bin\python3" C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py --configure C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/ --gen_root_path C:/users/<user>/_bazel_<user>/7cok3vt4/external/local_config_git/gen): The system cannot find the file specified.
 (error: 2)
ERROR: C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/core/BUILD:2494:1: @org_tensorflow//tensorflow/core:version_info_gen depends on @local_config_git//:gen/head in repository @local_config_git which failed to fetch. no such package '@local_config_git//': Traceback (most recent call last):
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 64
                _fail(result.stderr)
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
                fail(<1 more arguments>)
Git Configuration Error: java.io.IOException: ERROR: src/main/native/windows/process.cc(199): CreateProcessW("C:\users\<user>\_bazel_<user>\7cok3vt4\external\local_config_git\usr\bin\python3" C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py --configure C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/ --gen_root_path C:/users/<user>/_bazel_<user>/7cok3vt4/external/local_config_git/gen): The system cannot find the file specified.
 (error: 2)
ERROR: C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/core/BUILD:2494:1: @org_tensorflow//tensorflow/core:version_info_gen depends on @local_config_git//:gen/branch_ref in repository @local_config_git which failed to fetch. no such package '@local_config_git//': Traceback (most recent call last):
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 64
                _fail(result.stderr)
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
                fail(<1 more arguments>)
Git Configuration Error: java.io.IOException: ERROR: src/main/native/windows/process.cc(199): CreateProcessW("C:\users\<user>\_bazel_<user>\7cok3vt4\external\local_config_git\usr\bin\python3" C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py --configure C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/ --gen_root_path C:/users/<user>/_bazel_<user>/7cok3vt4/external/local_config_git/gen): The system cannot find the file specified.
 (error: 2)
ERROR: Analysis of target '//tensorflow_serving/model_servers:tensorflow_model_server' failed; build aborted: no such package '@local_config_git//': Traceback (most recent call last):
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 64
                _fail(result.stderr)
        File "C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
                fail(<1 more arguments>)
Git Configuration Error: java.io.IOException: ERROR: src/main/native/windows/process.cc(199): CreateProcessW("C:\users\<user>\_bazel_<user>\7cok3vt4\external\local_config_git\usr\bin\python3" C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py --configure C:/users/<user>/_bazel_<user>/7cok3vt4/external/org_tensorflow/ --gen_root_path C:/users/<user>/_bazel_<user>/7cok3vt4/external/local_config_git/gen): The system cannot find the file specified.
 (error: 2)
INFO: Elapsed time: 5.405s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (5 packages loaded, 139 targets configured)
```

### Exact Steps to Reproduce
1. `git clone https://github.com/tensorflow/serving.git`
2. `cd serving`
3. `bazel build ...` (see in the description above)

**Comment by mpmischitelli** [comment]
I've managed to solve this particular issue by commenting out these lines from .bazelrc:
```
build --action_env PYTHON_BIN_PATH="/usr/bin/python3"
build --define PYTHON_BIN_PATH=/usr/bin/python3
```
These set the aforementioned `PYTHON_BIN_PATH` env variable which is later used in `git_configure.bzl`. I've then just created a powershell script which sets the variable to my python installation and then invokes the `bazel build` command.

Now I'm facing more issues, but at least this particular one seems solved.
